runtime/nvqir/custatevec/CuStateVecCircuitSimulator.cu

```diff
@@ -298,9 +298,14 @@ protected:
       // FIXME handle case where pointer is a device pointer
 
       // First allocation, so just set the user provided data here
-      HANDLE_CUDA_ERROR(cudaMemcpy(deviceStateVector, state,
-                                   stateDimension * sizeof(CudaDataType),
-                                   cudaMemcpyHostToDevice));
+      {
```

Comment:
Do you really need this block (`{ ... }`)?

Reply:
No, we don't. Fixed in https://github.com/NVIDIA/cuda-quantum/pull/1607/commits/96b1112502e95b34cec0c9271b05ffe4035667cf